client/src/reducers/local-storage.reducer.ts

```diff
@@ -1,48 +1,48 @@
+import { createReducer } from '@reduxjs/toolkit';
 import { HelpCard } from '../actions/help-card.actions';
 import { Persist } from '../actions/persist.actions';
 import { User } from '../actions/user.actions';
+import { UserActions } from '../slices/user.slice';
 import { Dict } from '../utility/typescript-utils';
 import { getLoggedInState } from '../utility/user-utils';
 
 // We need to be careful about what goes into local storage due to security concerns. So make sure not to add anything
 // that is clearly identifying of groups, units, or individuals, such as PII/PHI data, group name, etc.
-
-export interface LocalStorageState extends Record<string, unknown>{
+export interface LocalStorageState extends Record<string, any>{
   orgId?: number
   hideHelpCard?: Dict<boolean>
 }
 
 export const localStorageInitialState: LocalStorageState = {};
 
-export function localStorageReducer(state = localStorageInitialState, action: any): LocalStorageState {
-  switch (action.type) {
-    case User.Actions.Register.type: {
+export const localStorageReducer = createReducer(localStorageInitialState, builder => {
```

Comment:
Apparently we can still leverage `immer` when using the `createReducer` function. So each of these cases can probably be simplified a bit.

https://redux-toolkit.js.org/usage/usage-guide#simplifying-reducers-with-createreducer